### ISSUE TITLE
emmylua ls stops working, need LSPStart to resume

### ISSUE THREAD

**Issue by cova-fe** [EmmyLuaLs/emmylua-analyzer-rust#781]
I'm using emmylua on macos and neovim, with a very simple configuration (the default one that comes via lspconfig)
```lua
return {
  cmd = { 'emmylua_ls' },
  filetypes = { 'lua' },
  root_markers = {
    '.luarc.json',
    '.emmyrc.json',
    '.luacheckrc',
    '.git',
  },
workspace_required = false,
}
```

The problem that I'm seeing is the following: once neovim is started, emmylua detected errors or warnings are reported inline, just fine. 
however, any following change in the code does not cause any change in reported errors, either inline or in trouble window. Removing the cause of the error od adding new errors has no effect on the messages. Only restarting LSP fixes the problem but only temporarily. I had a look at emmylua and lsp logs but no error or warnings are reported.
I'm not 100% sure if I hit a bug or some configuration issue, but I'm running out of ideas on how the debug the issue, so any hint would be appreciated.

**Comment by CppCXY** [comment]
If you can provide me with a complete Neovim configuration that is stable and reproducible, along with the steps to operate it and the command-line parameters used, I will take a look tomorrow.

**Comment by cova-fe** [comment]
After much digging, removing plugins to create a reproducible setup, it seems that the problem was caused by folke/lazydev.nvim. I guess some interaction was happening under the hood. Removing lazydev fixed the issues.
Thanks for the answer!
